shared/src/components/checkbox_input.tsx

```diff
@@ -37,18 +37,28 @@ const containerStyle = style({
 const StyledLabel = withStyles({
   label: {
     ...INPUT_TEXT_STYLE,
-    marginRight: 0,
+    marginRight: '-6px',
   },
 })(FormControlLabel);
 
+const CustomColorCheckBox = withStyles({
+  root: {
+    padding: '5px',
+    '&$checked': {
+      color: COLORS.focus,
+    },
+  },
+  checked: {},
+})((props: CheckboxProps) => <Checkbox color="default" {...props} />);
+
 /** Funtional Component for Checkbox input fields */
 export function CheckboxInput(props: CheckboxInputProps) {
   const { label, checked, ...inputProps } = props;
   return (
     <div className={containerStyle}>
       <StyledLabel
         control={
-          <Checkbox
+          <CustomColorCheckBox
```

Comment:
I assume this _shouldn't_ have any noticeable affect on other extensions that may be using this component?

Reply:
Yes it's just a color change. But I checked and made sure no other extensions are using this component.

Reply:
Correct. Also it's not used in any other extension.